tools/code_format/check_format.py

```diff
@@ -353,6 +358,9 @@ def allowlistedForUnpackTo(file_path):
       "./source/common/protobuf/utility.cc", "./source/common/protobuf/utility.h"
   ]
 
+def denylistedForExceptions(file_path):
+    return (file_path in EXCEPTION_DENYLIST or isInSubdir(file_path, 'tools/testdata')) and \
+        not file_path.endswith(DOCS_SUFFIX)
```

Comment:
This was a workaround because `isInSubdir` will include all top-level files (like `STYLE.md`)....